lib/fs.js

```diff
@@ -196,10 +196,14 @@ function Stats(
   this.ino = ino;
   this.size = size;
   this.blocks = blocks;
-  this._atim_msec = atim_msec;
-  this._mtim_msec = mtim_msec;
-  this._ctim_msec = ctim_msec;
-  this._birthtim_msec = birthtim_msec;
+  this.atim_msec = atim_msec;
```

Comment:
I feel like snake_case is pretty uncommon in node, so should maybe be changed.. but `mtimeMsec` is kinda ugly, so I'm not going to suggest it.